rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -341,9 +328,6 @@ upgrade_ncf() {
   fi
 
   if [ -f ${CONFIGURATION_REPOSITORY}/ncf/ncf.conf ]; then
-    # 2.9.0 & 2.11.11, 3.0.5: Make sure that ncf uses the right logger bundles (_logger_default and logger_rudder)
-    STEP="Make sure that ncf uses the right logger bundles (_logger_default and logger_rudder)"
```

Comment:
I think the comment should stay, but in a modified form, reflecting what is done below:
Make sure that ncf uses the right logger bundles (_log_default and log_rudder)"